pkg/api/handlers/compat/containers.go

```diff
@@ -242,6 +248,10 @@ func WaitContainer(w http.ResponseWriter, r *http.Request) {
 	// /{version}/containers/(name)/wait
 	exitCode, err := utils.WaitContainer(w, r)
 	if err != nil {
+		if errors.Cause(err) == define.ErrNoSuchCtr {
```

Comment:
Is this necessary? Waiting on a container that has been removed by `--rm` will error now

Reply:
Do you think we should at least log it. Or just ignore it.